### PR TITLE
is that even a thing?

### PR DESCRIPTION
**Pull request by quarterto**
🐿 v2.8.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "scripts": {
     "lerna": "lerna",
     "postinstall": "lerna bootstrap",
-    "rebootstrap": "lerna clean --yes && lerna bootstrap",
+    "rebootstrap": "lerna clean --yes && lerna bootstrap --no-ci",
```
Comment (chee) [inline package.json]: is it!??!

Comment (quarterto) [reply package.json]: nope

**Comment by chee** [comment]
`--no-ci`

[source](https://github.com/lerna/lerna/blob/d7e33c6e422934a37302f5e78e226423e3abcf04/integration/lerna-bootstrap.test.js#L19)

**Comment by quarterto** [comment]
...huh. didn't work in my testing. maybe the [`isCI` check](https://github.com/lerna/lerna/blob/5b01599/core/cli/index.js#L58) is overriding that

forcing travis onto npm 5.6 is fine for now

**Pull request merged**